bindings/protocols/coap/index.html

```diff
@@ -282,7 +325,6 @@ <h3>Observing Resources</h3>
             </p>
             <p>
                 Support for observing resources can be indicating by including a <code>subprotocol</code> value of <code>cov:observe</code> in a form.
-                However, a consumer can always fall back to polling a resource if observing a resource is permanently or temporarily not possible.
```

Comment:
This line was removed in d271c16daa0d34a229762f0b72a80033cda665b2 because it is redundant with the following paragraph, but was added back in 97156717b5528b39ade19360ee4ff167fca0c2ec. I have removed it again in 3713e2d241b579d09ca99f5792c9bb2cdc67ea31.

index.html looks good to me. I trust that the other files have all the necessary changes. So 👍  from me.

Reply:
Perfect, thanks!